cpp/src/io/csv/csv_reader.cu

```diff
@@ -1180,11 +1185,11 @@ gdf_error launch_dataTypeDetection(
 	column_data_t* d_columnData) 
 {
 	int blockSize;		// suggested thread count to use
-	int minGridSize;	// minimum block count required
-	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, dataTypeDetection) );
+ 	int minGridSize;	// minimum block count required
+ 	CUDA_TRY( cudaOccupancyMaxPotentialBlockSize(&minGridSize, &blockSize, dataTypeDetection) );
 
-	// Calculate actual block count to use based on records count
-	int gridSize = (raw_csv->num_records + blockSize - 1) / blockSize;
+ 	// Calculate actual block count to use based on records count
+ 	int gridSize = (raw_csv->num_records + blockSize - 1) / blockSize;
```

Comment:
Since the previous PR was accepted, merge conflicts resulted in having to manually copy the previous changes back, hence the unintentional extra spaces. Will proceed to fix these trivial things.